spec/system/claims/view_a_users_details.rb

```diff
@@ -1,46 +0,0 @@
-require "rails_helper"
-
-RSpec.describe "View a users details", type: :system do
-  before do
-    setup_school
-    create_user
-    attach_user_to_school
-  end
-
-  scenario "I sign in as a support user and invite a user to a school" do
-    sign_in_as_support_user
-    visit_claims_support_school_users_page
-    verify_user_details
-  end
-
-  private
-
-  def setup_school
-    @school = create(:school, :claims, urn: "123456", name: "Test School")
-  end
-
-  def create_user
-    @user = create(:user, :claims, first_name: "Barry", last_name: "Garlow", email: "barry.garlow@gmail.com")
```

Comment:
And this needs to be something like `create(:cliams_user)`

Reply:
resolved in [3c7e68f](https://github.com/DFE-Digital/itt-mentor-services/pull/211/commits/3c7e68fdbea308e765f63187dba7fb6dbb46d2c7)